[docs] Reviewing and updating contributing section

heitortsergent opened the pull request:
This PR updates and reviews the contributing section of the docs. It's available in staging as well.

@fearenales @nuxlli can you guys please review this?

- [x] Beside the JS lints, add a session about .editorconfig;
- [x] As well as we need to add guidelines about libnss-resolver to a azk installation via source code, it's important to mention it when talking about running azk to test contributions;
- [x] Highlight and explain why Node.js isn't required and how it works inside azk;
- [x] Include into Testing session informations regarding integration tests and bats usage;
- [x] Reinforce that everytime a single change is made in package.json it's needed to re-generate npm-shrinkwrap.json

nuxlli commented:
Very good, @heitortsergent. But I'd include some points:

- Beside the JS lints, add a session about .editorconfig;
- As well as we need to add guidelines about `libnss-resolver` to a `azk` installation via source code, it's important to mention it when talking about running `azk` to test contributions;
- Highlight and explain why Node.js isn't required and how it works inside `azk`;
- Include into Testing session informations regarding integration tests and bats usage;
- Reinforce that everytime a single change is made in `package.json` it's needed to re-generate `npm-shrinkwrap.json`

heitortsergent commented:
@nuxlli I added your points to the first comment as a checklist.

Could you review the last commit I made (it's in English only), I added information about the .editorconfig, the `libnss-resolver` and `npm-shrinkwrap`.

I need some help in the "highlight and explain why node.js isn't require" part (I added just a little bit of information about it), and the "Testing".

heitortsergent commented:
@marcusgadbem @fearenales @gullitmiranda @saitodisse I finished making the changes, in PT and EN. Can you guys please review it? :dancer:

nuxlli commented on docs/content/pt-BR/contributing/README.md:
```diff
@@ -1,74 +1,268 @@
-Os erros são reportados criando-se novas [issues] pelo [github]. Por favor, antes de criar novas [issues] verifique se elas já não foram criadas previamente.
+Os erros são comunicados através da criação de novas [issues] no [GitHub][github]. Elas nos ajudam a corrigir erros que possam ter escapado no nosso processo de teste ou release.
 
+Antes de criar novas [issues] por favor, certifique-se de que não há outras similares já criadas. Se você encontrar uma semelhante, adicione um comentário :+1: nela, e também qualquer informação que possa ser diferente da pessoa que a abriu pela primeira vez, por exemplo: OS, `azk`, versão, etc.
```
I thought the phrase "diferente da pessoa que" was a little strange.

nuxlli commented on docs/content/pt-BR/contributing/README.md:
```diff
@@ -1,74 +1,268 @@
+
+Como mencionado na seção "Detalhes de implementação", o `azk` é escrito em [Node.js][node.js], e utiliza vários recursos do ES6. Uma coisa a notar é que, para ajudar no desenvolvimento e contribuir para o `azk`, você não precisa ter Node.js instalado em sua máquina.
+
+O `azk` possui uma versão do Node.js, que é o mesmo que usamos para testá-lo e desenvolvê-lo, dentro da pasta "`/lib/nvm`". Ele já vem instalado e empacotado quando você baixa o `azk` através de um gerenciador de pacotes, ou ele também é instalado caso você baixe o projeto do GitHub e execute `make` para construir seu binário. Isso significa que nós podemos:
```
The right of the folder path is `./lib/nvm`.

saitodisse commented:
LGTM in pt-br

heitortsergent commented:
@nuxlli made a new commit with the changes you mentioned :). Let me know if it looks good so I can merge. :dancer:

nuxlli commented:
LGTM @heitortsergent
